### PR TITLE
feat(FOROME-841): add checkbox and radio components 

### PR DESCRIPTION
**Pull request by QSdmitrioul**
https://quantori.atlassian.net/browse/FOROME-841

### CODE REVIEW

File: src/ui/check-input.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <label htmlFor={id} className={cn(className)}>
+      <input
+        type={type}
+        id={id}
+        checked={checked}
+        disabled={disabled}
+        onChange={onChange}
+        className={cn({
+          'mr-2': type === 'checkbox',
+          'mr-1': type !== 'checkbox',
```
Comment (AndreyUstyumenko) [inline src/ui/check-input.tsx]: type === 'checkbox' ? 'mr-2' : 'mr-1'

File: src/ui/check-input.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  id?: string
+}
+
+export const CheckInput: FC<ICheckInputProps> = ({
+  checked,
+  disabled,
+  onChange,
+  id,
+  className,
+  type = 'radio',
```
Comment (AndreyUstyumenko) [inline src/ui/check-input.tsx]: no default value

File: src/ui/check-input.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React, { ChangeEvent, FC } from 'react'
+import cn, { Argument } from 'classnames'
+
+export interface ICheckInputProps {
+  checked?: boolean
+  onChange?: (e: ChangeEvent<HTMLInputElement>) => void
+  className?: Argument
+  disabled?: boolean
+  type?: 'checkbox' | 'radio'
```
Comment (AndreyUstyumenko) [inline src/ui/check-input.tsx]: type is not optional

File: src/ui/check-input.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import React, { ChangeEvent, FC } from 'react'
+import cn, { Argument } from 'classnames'
+
+export interface ICheckInputProps {
+  checked?: boolean
```
Comment (AndreyUstyumenko) [inline src/ui/check-input.tsx:5]: checked is not optional

File: src/ui/radio.tsx [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import { FC } from 'react'
+
+import { CheckInput, ICheckInputProps } from '@ui/check-input'
+
+export const Radio: FC<Omit<ICheckInputProps, 'type'>> = props => (
+  <CheckInput {...props} />
```
Comment (AndreyUstyumenko) [inline src/ui/radio.tsx]: type="radio"

File: src/ui/checkbox/checkbox.css [hunk 6]
```diff
@@ -0,0 +1,27 @@
+input[type="checkbox"] {
```
Comment (AndreyUstyumenko) [inline src/ui/checkbox/checkbox.css]: make styling with styled-components

**Review by QYaroslavTrefilov — APPROVED** [review]
I think "cursor: pointer" should be added for labels.

**Comment by AndreyUstyumenko** [comment]
> I think "cursor: pointer" should be added for labels.

I don't think so, it's quite rare behavior in UX

**Review by AndreyUstyumenko — APPROVED** [review]

**Pull request merged**